### ISSUE TITLE
Add `participated_in` for Person

### ISSUE THREAD

**Issue by sdevalk** [linked-art/linked.art#453]
Hi all,

I'm using Linked Art for modelling metadata of works of art - and making nice progress, thanks to the documentation on the LA website!

I also want to model the metadata of persons that are depicted on these works of art. I can use the [Actor Model](https://linked.art/model/actor/) as a foundation. However, the data contains rather specific life activities - such as 'baptism', 'engagement', 'marriage' and 'burial' - that are, I think, out of scope of LA. Do you recommend certain models or best practices that I can use for modelling biographical data?

Thank you.

(I'm not sure if this repository is meant for Q&A - feel free to close this issue.)

**Comment by azaroth42** [comment]
Hi there!

Activities are relatively easy (compared to relationships, which are much harder).

You could use `carried_out` for activities other than the profession activities that museums typically care about, and do something like:

```
{
  "id": "...",
  "type": "Person",
  "carried_out": [
    {"type": "Activity", "classified_as": [{"id": "aat:300393188", "type": "Type", "_label": "engagement"}], ...}
  ]
}
```

That doesn't help for burial or baptism, where it would mean that they did the burying or baptizing, rather than being buried/baptised, but you could use `participated_in` / `crm:P11i_participated_in` (which isn't documented in linked art but we could add it) instead of `carried_out`.

**Comment by sdevalk** [comment]
Hi Rob,

Thank you! `carried_out` and `participated_in` make sense.

About `participated_in`: does this sufficiently express that the activity is about the person being baptized or buried? Or is there a way to make clear that the person has a certain 'role' in this activity?

About `carried_out`, when used in an 'engagement' or 'wedding': is it possible to establish a relationship here, i.e. that this person married some other person?

**Comment by azaroth42** [comment]
To all of your very excellent questions ... No ... but it's what we've got :(

Arcana follows!

TLDR:  You could partition an event to be more specific about roles ... or you could just make some blanket assumptions and be right 99% of the time.

CRM gives us:
* P12 - occurred in the presence of:  This is just physically present, but not actively participating
* P11 - had participant: Physically present, and either passively or actively participating, but not "carrying out" (e.g. a sitter participated in the creation of the portrait, but did not carry it out, unless it was a self-portrait)
* P14 - carried out by: Active participation with some sort of causal or legal responsibility. The activity would not have occurred without the actor that carries it out.

Now, there is also P16 - used specific object which "describes the use of material or immaterial things in a way essential to the performance of the outcome of an instance of E7 Activity." Which people are material things, so does the baptism activity use the child as a thing? The scope note goes on: "This property typically applies to tools, instruments, moulds, raw materials, and items embedded in a product." There is the weasel word of "typically", but the intent is clearly for inanimate objects. 

What about role? In pure CRM there are .1 properties to describe role or type, but there isn't for P11, only for P14. Regardless, we don't use .1 in linked art, as it is incompatible with RDF without reification of the relationship, and we have other patterns that are just as good... like ... partitioning.

So, if you want to go into detail about the baptism / engagement / marriage / burial, you could turn it into a full Activity entry (https://linked.art/api/1.0/endpoint/event/) and then partition out the roles into components. Messy ... but less messy than creating dozens of 'baptized_by', 'bride_during', 'buried' type of relationships.

Something like:

```
{
  "@context": "https://linked.art/ns/v1/linked-art.json",
  "id": "http://lod.example.org/museum/Activity/0",
  "type": "Activity",
  "classified_as": [
    {
      "id": "http://vocab.getty.edu/aat/300069030",
      "type": "Type",
      "_label": "Baptism"
    }
  ],
  "carried_out_by": [
    {
      "id": "http://lod.example.org/museum/Person/1",
      "type": "Person",
      "_label": "Priest"
    }
  ],
  "part": [
    {
      "id": "http://lod.example.org/museum/Activity/1",
      "type": "Activity",
      "classified_as": [
        {
          "id": "http://example.org/baptisee",
          "type": "Type",
          "_label": "Being Baptized"
        }
      ],
      "participant": [
        {
          "id": "http://lod.example.org/museum/Person/0",
          "type": "Person",
          "_label": "Baby"
        }
      ]
    }
  ]
}
```

Which is still mediocre. The "part" relationship is P9 consists of, which says in its scope note that the part is "defined by a subset of the phenomena that define the [whole]". The baby's participation in the baptism is a subset of the phenomena, compared to the subset of phenomena carried out by the priest or parents or congregation, for example. 

However ... it seems like the only participant in the activity that we really care about is the one being baptized, and possibly the priest carrying it out. Which doesn't need all of this additional machinery, beyond carried_out and participated_in.

Ditto the wedding / married, engagement / engaged pattern. It means quite a lot of hoops to be clear about bride vs groom vs flowergirl vs best man etc. when mostly we care about A and B getting married, and ceremony carried out by C. The establishment of the A is-married-to B relationship is a Phase [see #180 and related] which we've avoided going to so far, as it causes no end of headaches. To summarize a long set of discussions:  A is-married-to B is only true in some context, for some period of time. The marriage may not be recognized as legal in some jurisdictions, and if they were to get divorced, then it would have a start and end date. We recognize the benefit of having such a capability of description ... but it's not critical to linked *art* as opposed to a corresponding linked *people* spec.

Thoughts? :)

**Comment by sdevalk** [comment]
Hi Rob,

Thanks for your thorough explanation - much appreciated!

Yes, I get your point about very specific yet unwieldy modelling versus using a more straightforward approach that's easier to understand although it not always captures the full meaning of the data. If I follow the straightforward approach (e.g. by using `participated_in` and `carried_out` for 'baptism', 'engagement', 'marriage' and 'burial') then I'm only unable to express that person A was engaged/married to person B.

I agree, this topic - modelling specific life events and relationships of people - isn't something that LA should tackle. Do you know of a model/spec that I could use for this? For example: I know of Bio CRM, but I am not sure about its uptake - is it being used?

**Comment by azaroth42** [comment]
There's a working group in the CRM SIG called CRMSoc (Social) which is slowly, slowly working on these sorts of things: https://cidoc-crm.org/crmsoc/home-10 They haven't released any actual extension yet.

**Comment by sdevalk** [comment]
Hi Rob, thank you!

**Comment by azaroth42** [comment]
Reopening and tagging as model -- should we add `participated_in` to the Person model, or leave it as a CRM-compatible extension?

**Comment by azaroth42** [comment]
e.g. https://deploy-preview-459--linked-art.netlify.app/model/actor/#participation-in-events

**Comment by azaroth42** [comment]
Discussion on call 2022-01-12:  Can we just use `carried_out_by` instead of `participated_in`? If we understand "causal" as "necessary participation" then the baby is causal for the baptism, and not just a passive bystander. Clearly engagement and marriage can be carried_out_by. Technically the person-as-dead-body is P12 present at the burial, but (per the badly rendering note in /model/actors/) we don't instantiate a new entity to represent the "Physical Thing" of the body, separate from the Person... otherwise we'd need to do that for all dead "people".  So following that, plus the necessary = causal, the person can carry out their own burial.

Have emailed the SIG list to ask about the intent of "causal" here.

**Comment by azaroth42** [comment]
From Martin on the SIG list:

> The role of the child should definitely not be regarded as P14. Necessary participation is definitely not meant by "causal" or "legal responsibility" in this scope note. P14 is meant in an active sense. In a sense, any participation may have an impact on an event. This was not meant here. Similarly, patients in a surgery are necessary, but do not carry it out.

So we would need to add one of `participated_in` and `present_at`. Given the use case is biographical, I think `participated_in` is the right one to add.

Homework for everyone remains to find other situations where such passive or non-causal participation of actors in events exists.

**Comment by azaroth42** [comment]
```
"participated_in": [{
  "type": "Activity",
   "classified_as": [{
       "id": "http://vocab.getty.edu/aat/baptism" // aat:going-on-holiday, ...
       "type": "Type",
    ...
   }]
 }]
```

**Comment by azaroth42** [comment]
Discussion on call 2022-06-01: agree that `participated_in` as above should be added
